platforms/documentation/docs/src/docs/userguide/dep-man/dependency-management/component_metadata_rules.adoc

```diff
@@ -343,18 +310,16 @@ However, modifying attributes or capabilities on Ivy configurations has no effec
 For very Ivy specific use cases, the component metadata rules API also offers access to other details only found in Ivy metadata.
 These are available through the link:{javadocPath}/org/gradle/api/artifacts/ivy/IvyModuleDescriptor.html[IvyModuleDescriptor] interface and can be accessed using `getDescriptor(IvyModuleDescriptor)` on the link:{javadocPath}/org/gradle/api/artifacts/ComponentMetadataContext.html[ComponentMetadataContext].
 
-.Ivy component metadata rule
 ====
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/kotlin",files="build.gradle.kts[tags=ivy-component-metadata-rule]"]
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/groovy",files="build.gradle[tags=ivy-component-metadata-rule]"]
 ====
 
-== Filter using Maven metadata  ==
+== Filter using Maven metadata
 
 For Maven specific use cases, the component metadata rules API also offers access to other details only found in POM metadata.
 These are available through the link:{javadocPath}/org/gradle/api/artifacts/maven/PomModuleDescriptor.html[PomModuleDescriptor] interface and can be accessed using `getDescriptor(PomModuleDescriptor)` on the link:{javadocPath}/org/gradle/api/artifacts/ComponentMetadataContext.html[ComponentMetadataContext].
 
-.Access pom packaging type in component metadata rule
```

Comment:
🤔 Initially saying "or a more mature status." sounds weird - it would help to have the list of statuses earlier in these paragraphs to give this more context.

"in the components status scheme" should be "in the component's status scheme".

"with the state integration" should be "with the STATUS integration"

Reply:
Updated all items.